package day42_Exceptions;

public class ThrowsKeyword {

    public static void main(String[] args) throws InterruptedException {


        System.out.println("---------------Test 1-----------------");

        System.out.println("Hello");

        Thread.sleep(1500);

        System.out.println("Cydeo");

        System.out.println("--------------------- Test 1 completed-----------");

        System.out.println("---------------Test 2-----------------");

        System.out.println("Hello");

        Thread.sleep(1000);

        System.out.println("Batch 25");

        System.out.println("--------------------- Test 2 completed-----------");

    }


}
